gradle/libs.versions.toml

```diff
@@ -33,6 +33,7 @@ dependencyGuard = "0.4.3"
 firebaseBom = "32.4.0"
 firebaseCrashlyticsPlugin = "2.9.9"
 firebasePerfPlugin = "1.4.2"
+firebaseTestlab = "0.0.1-alpha05"
```

Comment:
Just a warning, if you ever need to bump this to alpha06 or above, it requires AGP 8.3. mentioning because of previous hesitancy to use unstable releases.